### PR TITLE
TOOL-527 Added case to handle optional values.

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**

### CODE REVIEW

File: stepconf/stepconf.go [hunk 1]
```diff
@@ -195,15 +195,18 @@ func ValidateRangeFields(valueStr, constraint string) error {
...
 	if err != nil {
-		return fmt.Errorf("failed to parse min value: %s", err)
+		return fmt.Errorf("failed to parse min value %s: %s", constraintMin, err)
 	}
 	max, err := parseValueStr(constraintMax)
 	if err != nil {
-		return fmt.Errorf("failed to parse max value: %s", err)
+		return fmt.Errorf("failed to parse max value %s: %s", constraintMax, err)
+	}
+	if valueStr == "" {
```
Comment (trapacska) [inline stepconf/stepconf.go]: Please see my comment.

Comment (richard-bogdan-bitrise) [reply stepconf/stepconf.go]: Updated code.

Comment (trapacska) [reply stepconf/stepconf.go]: please move this empty check to the first line of the block to exit-fast

Comment (richard-bogdan-bitrise) [reply stepconf/stepconf.go]: Moved to first line.

**Pull request merged**